src/qml/filters/fisheye/ui.qml

```diff
@@ -0,0 +1,827 @@
+            id: preset
+            parameters: defaultParameters
+            onPresetSelected: setControlData()
+        }
+        Row {width: 6} //blank
+
+        // Row 2 defish/fish
+        Label {
+            text: qsTr('Fisheye')
+            Shotcut.HoverTip { text: qsTr('Add or remove fisheye effect') }
+            Layout.alignment: Qt.AlignRight
+        }
+        RowLayout {
+            ButtonGroup { id: fisheyeGroup }
+            RadioButton {
+                id: fisheyeRemoveButton
+                text: qsTr('Remove')
+                ButtonGroup.group: fisheyeGroup
+                checked: deFishDefault
+                onCheckedChanged: {
+                    if (blockUpdate) return
+                    if (checked) { filter.set(deFishParam, true) }
+                }
+            }
+            RadioButton {
+                id: fisheyeAddButton
+                text: qsTr('Add')
+                ButtonGroup.group: fisheyeGroup
+                checked: !deFishDefault
+                onCheckedChanged: {
+                    if (blockUpdate) return
+                    if (checked) { filter.set(deFishParam, false) }
+                }
+            }
+        }
+        Shotcut.UndoButton {
+            onClicked: {
+                fisheyeRemoveButton.checked = deFishDefault
+                fisheyeAddButton.checked = !deFishDefault
+            }
+        }
+
+        // Row 3: focal ratio
+        Label {
+            text: qsTr('Focal ratio')
+            leftPadding: 10
+            Shotcut.HoverTip { text: qsTr('The amount of lens distortion') }
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.SliderSpinner {
+            id: focalRatioSlider
+            minimumValue: 0
+            maximumValue: 1
+            decimals: 3
+            suffix: ''
+            value: focalRatioDefault
+            onValueChanged: { if (blockUpdate) return; filter.set(focalRatioParam, value) }
+        }
+        Shotcut.UndoButton {
+            onClicked: focalRatioSlider.value = focalRatioDefault
+        }
+
+        // Row 4: Interpolation Quality
+        Label {
+            text: qsTr('Quality')
+            Shotcut.HoverTip { text: qsTr( 'Resample quality') }
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.ComboBox {
+            id: qualityCombo
+            implicitWidth: 120
+            model: ListModel {
+                id: qualityModel
+                ListElement { text: qsTr('1. Nearest neighbor'); value: 0.0 }
+                ListElement { text: qsTr('2. Bilinear'); value: 0.166 }
+                ListElement { text: qsTr('3. Bicubic smooth'); value: 0.333 }
+                ListElement { text: qsTr('4. Bicubic sharp'); value: 0.500 }
+                ListElement { text: qsTr('5. Spline 4x4'); value: 0.666 }
+                ListElement { text: qsTr('6. Spline 6x6'); value: 0.833 }
+                ListElement { text: qsTr('7. Lanczos 16x16'); value: 1.0 }

```

Comment:
Nowhere else in the Shotcut UI do we number the combobox list items. It might be your preference, but these should not be numbered to be consistent.